src/pg/pg-store.ts

```diff
@@ -98,12 +104,12 @@ export class PgStore extends BasePgStore {
                 },
                 location: {
                   block_hash,
+                  block_height,
                   tx_id,
                   genesis_id: reveal.inscription_id,
-                  block_height: event.block_identifier.index,
                   address: reveal.inscriber_address,
-                  output: `${tx_id}:0`,
-                  offset: reveal.ordinal_offset.toString(),
+                  output: `${satpoint.tx_id}:${satpoint.vout}`,
+                  offset: satpoint.offset ?? null,
```

Comment:
null or 0?

Reply:
IIRC chainhooks returns this as null if the satpoint gets spent as fees? Or do I have that wrong?

Reply:
Oh you're right, perfect. That works, but we'll revisit this behavior chainhook side. Thanks!